src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -295,21 +295,26 @@ private static PackageReference ToPackageReference(LibraryDependency library, Nu
                 .GetPackageReferencesAsync(targetFramework, CancellationToken.None))
                 .ToList();
 
-            var packageTargetFallback = _vsProjectAdapter.PackageTargetFallback?.Split(new[] { ';' })
+            var splitChars = new[] { ';' };
```

Comment:
I'm mixed on this. I'm not sure that the benefit of making this static out weighs creating this every time the assembly is loaded.